node/deepgram-transcribe-video/index.js

```diff
@@ -0,0 +1,66 @@
+const axios = require("axios").default;
+
+module.exports = async (req, res) => {
+  let fileUrl;
+  try {
+    const payload = JSON.parse(req.payload);
+    fileUrl = payload.fileUrl;
+  } catch (err) {
+    res.json({
+      success: false,
+      message: "Payload does not have a proper structure.",
+    });
+    return;
+  }
+  let deepgramApiKey = req.variables["DEEPGRAM_API_KEY"];
```

Comment:
FYI, best practice is to use `const` rather than `let` for variables that don't change.